gradle-palantir-java-format/src/main/groovy/com/palantir/javaformat/gradle/ConfigureJavaFormatterXml.groovy

```diff
@@ -30,9 +30,11 @@ class ConfigureJavaFormatterXml {
             listItems.appendNode('option', [value: uri])
         }
         // configure nativeImageClasspath
-        nativeImageUri.ifPresent { URI uri ->
+        nativeImageUri.ifPresentOrElse({ URI uri ->
```

Comment:
if we remove the gradle property `palantir.native.formatter`, intellij needs to switch to using the bootstrapping jdk. therefore, we need to delete the nativeImageClassPath entry in the `palantir-java-format.xml` file. Fixes issue introduced in: https://github.com/palantir/palantir-java-format/pull/1200/commits/1f084a1592bc9b2ebbaf1ffcb3b26d73bb6aec33#diff-a2d2407a6a459747eaf9b40ec418e6ac7824c159c005b1f173623906db71a336L75

Reply:
moved to a different PR: https://github.com/palantir/palantir-java-format/pull/1217